src/main/java/org/broadinstitute/hellbender/tools/exome/conversion/allelicbalancecaller/AllelicSplitCall.java

```diff
@@ -1,29 +1,29 @@
-package org.broadinstitute.hellbender.tools.exome.cnlohcaller;
+package org.broadinstitute.hellbender.tools.exome.conversion.allelicbalancecaller;
 
 import htsjdk.samtools.util.Locatable;
 import org.broadinstitute.hellbender.tools.exome.ACNVModeledSegment;
 
 import java.io.Serializable;
 
-public class CNLOHCall implements Serializable, Locatable{
+public class AllelicSplitCall implements Serializable, Locatable {
```

Comment:
I'm just going to do AllelicCalls.  I don't want to spend any more time on this.